docs/GettingStarted/QuickstartTutorial.md

```diff
@@ -1,280 +1,366 @@
 
-## Mean Time to Restore and Change Failure Rate
+### Mean Time to Restore and Change Failure Rate
 
 In this section you see how Pelorus measures the last two DORA metrics: **Mean time to recovery** and **Change failure rate**.
 
 ![dora_fail_recover](../img/dora_metrics2.png)
 
-It is necessary to have Github issues enabled in your fork of the mig-apps-demo repository (visit settings to enable it).
-
-### Pelorus configuration
+#### Pelorus failure exporter
 
-After following [Lead Time for Change and Deployment Frequency](#lead-time-for-change-and-deployment-frequency), update the **exporters** section of the `/var/tmp/values.yaml` to match the following example:
-```yaml
-[...]
-exporters:
-  instances:
-  - app_name: deploytime-exporter
-    exporter_type: deploytime
-    extraEnv:
-    - name: LOG_LEVEL
-      value: DEBUG
-    - name: NAMESPACES
-      value: mongo-persistent
-  - app_name: committime-exporter
-    exporter_type: committime
-    extraEnv:
-    - name: LOG_LEVEL
-      value: DEBUG
-    - name: NAMESPACES
-      value: mongo-persistent
-  - app_name: failure-exporter
-    exporter_type: failure
-    extraEnv:
-    - name: LOG_LEVEL
-      value: DEBUG
-    - name: PROVIDER
-      value: github
-    - name: TOKEN
-      value: <TOKEN>
-    - name: PROJECTS
-      value: <your_org>/mig-demo-apps
-```
-
->**Note:**
->
-> * A user [Github personal access token](https://github.com/settings/tokens) is required.
-> * The `PROJECTS` key's value is the fork of the mig-apps-demo repository.
+[Creating Pelorus instance](#creating-pelorus-instance) step should gave you a running instance of the `failure` exporter.
 
-Apply the updated values for Pelorus by executing:
-```
-helm upgrade pelorus charts/pelorus --namespace pelorus --values /var/tmp/values.yaml
+Check that no bugs are found by Pelorus in its' `failure` exporter, by ensuring no lines starting with the `failure_creation_timestamp` are present in the output of the following command:
+```shell
+$ curl $(oc get route failure-exporter --namespace pelorus -o=template='http://{{.spec.host | printf "%s\n"}}')
 ```
 
-Wait for the upgrade to complete.
+#### GitHub Issues
 
-Check the output from the failure exporter, by running
-```
-curl $(oc get route failure-exporter --namespace pelorus -o=template='http://{{.spec.host | printf "%s\n"}}')
-```
-No bugs should be found at this time.
+##### Prepare GitHub labels
+Pelorus will utilize two labels to determine if a GitHub issue is associated with our `todolist` mongo application. Only those GitHub issues which have **both** labels are recognized by Pelorus as a critical bugs in our "production" environment:
```

Comment:
add the text 

Only those GitHub issues which have **both** labels are recognized by Pelorus as a critical bugs in our "production" environment

after the bullets

Reply:
changed.

Reply:
also moved this as a Note.